src/store/item.js

```diff
@@ -1,4 +1,13 @@
 import { writable } from 'svelte/store';
+import colorOptions from '@/constants/item/colors';
+
+export const colors = writable(colorOptions);
+export const customColors = writable({});
```

Comment:
Why do we need 2 separate structures?

Reply:
They will be merged after product creation. As I wrote in the previous comment, it distinguishes between pre-defined colors and colors for varieties without mixing indexes.